deploy/dynamo/sdk/src/dynamo/sdk/cli/serve.py

```diff
@@ -155,10 +155,11 @@ def deep_update(d: dict, key: str, value: t.Any):
 
 
 def build_serve_command() -> click.Group:
-    from bentoml._internal.log import configure_server_logging
     from bentoml_cli.env_manager import env_manager
     from bentoml_cli.utils import AliasCommand
 
+    from dynamo.sdk.lib.logging import configure_server_logging
```

Comment:
```suggestion
    from dynamo.sdk.lib.logging import configure_logging
```

Question: why "sever". should this be "service" or could be just configure logging?

Reply:
I tried to match one of bento's APIs here. They used to have this concept of `configure_server_logging` so wanted to make it drop in for now